doc/release-notes.md

```diff
@@ -28,22 +35,366 @@ reindex or re-sync the whole chain.
+- The `connections` field of `dash-cli -getinfo` is expanded to return a JSON
+  object with `in`, `out` and `total` numbers of peer connections and `mn_in`,
+  `mn_out` and `mn_total` numbers of verified mn connections. It previously
+  returned a single integer value for the total number of peer connections. (dash#5823)
+- Update `-getinfo` to return data in a user-friendly format that also reduces vertical space.
+- A new CLI `-addrinfo` command returns the number of addresses known to the
+  node per network type (including Tor v2 versus v3) and total. This can be
+  useful to see if the node knows enough addresses in a network to use options
+  like `-onlynet=<network>` or to upgrade to current and future Tor releases
+  that support Tor v3 addresses only.  (#5904)
+- CLI `-addrinfo` now returns a single field for the number of `onion` addresses
+  known to the node instead of separate `torv2` and `torv3` fields, as support
+  for Tor V2 addresses was removed from Dash Core in 18.0.
+
+dash-wallet changes
+-------------------
+This release introduces several improvements and new features to the `dash-wallet` tool, making it more versatile and
+user-friendly for managing Dash wallets.
+
+- Wallets created with the `dash-wallet` tool will now utilize the `FEATURE_LATEST` version of wallet which is the HD
+  (Hierarchical Deterministic) wallets with HD chain inside.
+- new command line argument `-usehd` allows creation of non-Hierarchical Deterministic (non-HD) wallets with the
+  `wallettool` for compatibility reasons since default version is bumped to HD version
+- new command line argument `-descriptors` enables _experimental_ support of Descriptor wallets. It lets users
+  create descriptor wallets directly from the command line. This change aligns the command-line interface with
+  the `createwallet` RPC, promoting the use of descriptor wallets which offer a more flexible ways to manage
+  wallet addresses and keys.
+- new commands `dump` and `createfromdump` have been added, enhancing the wallet's storage migration capabilities. The
+  `dump` command allows for exporting every key-value pair from the wallet as comma-separated hex values, facilitating a
+  storage agnostic dump. Meanwhile, the `createfromdump` command enables the creation of a new wallet file using the
+  records specified in a dump file. These commands are similar to BDB's `db_dump` and `db_load` tools and are useful 
+  for manual wallet file construction for testing or migration purposes.
 
-A deadlock caused nodes to become non-responsive and RPC to report "Work depth queue exceeded".
-Thanks to Konstantin Akimov (knst) who discovered the cause. This previously caused masternodes to become PoSe banned.
 
-# v20.1.1 Change log
+# v21.0.0 Change log
 
 See detailed [set of changes][set-of-changes].
 
 # Credits
 
 Thanks to everyone who directly contributed to this release:
 
+- Kittywhiskers Van Gogh
 - Konstantin Akimov
 - PastaPastaPasta
 - thephez
```

Comment:
missing panleone. 

See https://github.com/dashpay/dash/commit/72d2008901b28d4761cd7543ca60ba441e42c027 and several more commits.